client/src/lib/dependency-injection/injectGraphql.js

```diff
@@ -31,14 +31,9 @@ const injectGraphql = (key, context) => (DataHandler) => {
           const apolloHOC = graphqlContainer.getApolloHOC();
           target = apolloHOC(DataHandler);
           error = false;
-        } catch (e) {
+        } finally {
           this.setState({ target, error });
-
-          // re-throw the error, as we do not want to silence it in the console
-          throw e;
         }
-
-        this.setState({ target, error });
```

Comment:
This is unrelated to the rest of the changes and I'll remove it if you want - but it's just a lot cleaner and results in a stack trace from where the exception originated rather than pointing HERE as the origin of the exception.

Basically, it used to be
```js
try {
  doSomething();
} catch (e) {
  finish();
  throw e;
}
finish();
```
and now it's
```js
try {
  doSomething();
} finally {
  finish();
}
```
which does the same thing but results in a more accurate stack trace.